src/main/webapp/WEB-INF/pages/registration.jsp

```diff
@@ -0,0 +1,29 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Registration</title>
+    <link href="/resources/styles/registration.css" rel="stylesheet"/>
+</head>
+<body>
+<div>
+    <form id="registrationForm" class="form" action="/registration" method="post">
+        <h2>Registration</h2>
+        <label for="userName">Login</label>
+        <input id="userName" name="userName" type="text"/>
+
```

Comment:
I ain't good at html css, but I think that empty lines between tags should have been deleted by you